controllers/articles.controller.js

```diff
@@ -15,8 +16,11 @@ exports.getTopics = (req, res, next) => {
 
 exports.getArticlesById = (req, res, next) => {
   const id = req.params.article_id;
-  fetchArticleById(id)
-    .then((articles) => {
+  Promise.all([fetchArticleById(id), fetchCommentsByArticleId(id)])
```

Comment:
You are contacting the database twice here - I would suggest taking a look at at _**SQL Relationships**_ notes for a hint on how to use SQL Joins to add complexity to your query, but get all your necessary information in one go